index.bs

```diff
@@ -2224,53 +2207,42 @@ The player responds to `createSession` with a `resolve` message.
 
 ### Session Establishing Delays and Failures
```

Comment:
My feeling is this section is all implementation details (and strange edge case implementation details - like a bug in the SIMID creative). Personally I would just compress it to one sentence. 

Something like:
If the SIMID creative fails to call createSession, then the player may play the media and report error 1212.

Remember, the whole point of 1.0.1 is to make things more simple and readable.